raft/doc.go

```diff
@@ -257,6 +257,12 @@ stale log entries:
 	If candidate receives majority of votes of denials, it reverts back to
 	follower.
 
+  'MsgPreVote' and 'MsgPreVoteResp' are used in an optional two-phase election
```

Comment:
Fixed, I think. It's difficult to distinguish between tabs and one tab's worth of spaces.